paxml/trainer_lib.py

```diff
@@ -1142,6 +1149,13 @@ def train_step_single_learner(
         mdl_vars,
         vars_with_opt,
     )
+    # Update 'overwrite_with_gradient' params.
+    mdl_vars = jax.tree_map(
+        lambda e, new, old: new if e else old,
```

Comment:
can we pass hparams here and use the function (see other comment)

Reply:
Yes, this is removed.